src/ballet/pack/BUILD

```diff
@@ -1,17 +1,34 @@
-load("//bazel:fd_build_system.bzl", "fd_cc_library", "fd_cc_test")
+load("//bazel:fd_build_system.bzl", "fd_cc_library", "fd_cc_test", "fd_cc_binary")
+
+package(default_visibility = ["//src/ballet:__subpackages__"])
 
 fd_cc_library(
     name = "pack",
+    srcs = [ "fd_pack.c" ],
     hdrs = [
+        "fd_pack.h",
         "fd_compute_budget_program.h",
         "fd_est_tbl.h",
     ],
     visibility = ["//visibility:public"],
     deps = [
+        "//src/tango",
+        "//src/ballet/txn",
+        "//src/ballet/ed25519",
+        "//src/ballet/poh",
+        "//src/ballet/shred",
+        "//src/ballet/bmtree",
```

Comment:
It doesn't actually depend on these, but I had to add them to get it to compile. Not sure if there's a better way.